testutil/dummy.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package testutil
+
+import "github.com/ethereum/go-ethereum/p2p"
+
+// DummyMsgRW implements MessageReader and MessageWriter
+// but doesn't do anything. Useful for dummy message sends
+type DummyMsgRW struct{}
```

Comment:
not sure why u need this file at all...

Reply:
It adds some dummy objects separately, it could go into an existing file but why not keep them separate

Reply:
if we need this, there is something wrong with testing

Reply:
It was already existing and I just made it public. Why would there be something wrong with testing? Using `protocolTester` for swap is virtually impossible, as the new Handshakes involved require contract addresses which require deployment of contracts which make tests unnecessarily complex in some situations

Reply:
This actually had to be reverted due to a recent commit to `master` by @acud in the `testutil` package. We had not rebased since, thus this introduced a circular dependency. 

So removed the file and reverted to original state